mmcls/models/backbones/mlpmixer.py

```diff
@@ -0,0 +1,51 @@
+from torch import nn
+from einops.layers.torch import Rearrange
+
+from ..builder import BACKBONES
+from .base_backbone import BaseBackbone
+from .vision_transformer import FFN
+
+class MixerBlock(nn.Module):
```

Comment:
Now, we use `mmcv.runner.BaseModule` to replace `nn.Module`, and it supports `init_cfg` to specify initialization function of layers. Please use it.